.travis.yml

```diff
@@ -2,6 +2,9 @@ language: python
 
 python:
   - "2.7"
+  - "3.3"
+  - "3.4"
+  - "3.5"
```

Comment:
Travis recently added 3.6 to their support matrix: https://github.com/travis-ci/travis-ci/issues/7085